.github/workflows/crowdin-pull.yml

```diff
@@ -0,0 +1,26 @@
+name: Pull Translations from Crowdin
+
+on:
+  schedule:
+    - cron: '0 0 * * 0' # Runs weekly at midnight on Sunday
+  workflow_dispatch: # Allows manual triggering of the workflow
+
+jobs:
+  pull-translations:
+    runs-on: ubuntu-latest
```

Comment:
avoid using latest, pinned versions are safer for any potential future breakages